aiida_pseudo/cli/params/options.py

```diff
@@ -82,3 +82,7 @@
     show_default=True,
     help='Specify the energy unit of the cutoffs. Must be recognized by the ``UnitRegistry`` of the ``pint`` library.'
 )
+
+DOWNLOAD_ONLY = OverridableOption(
+    '--download-only', is_flag=True, help='Only download the pseudopotential files and do install as a family.'
```

Comment:
I suppose you mean that they can use `aiida-pseudo install family` after the download to install the pseudo family? I'm not sure if that is clear from this message. I'd make it something like:

```suggestion
    '--download-only', is_flag=True, help='Only download the pseudopotential files to the current working directory, without installing the pseudopotential family.'
```